0.12.4/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM hashicorp/terraform:0.12.4
+
+ENV TFNOTIFY_VERSION=0.3.0
+
+RUN    apk add --update --no-cache --virtual .build-deps curl \
```

Comment:
blank

Reply:
It seems that the lint for align the start with the command.